src/lib/components/LanguageSelector.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+    import { locale } from '../../i18n/i18n-svelte';
+
+    let languages = [
+        { lng: 'select language', value: window.localStorage.getItem('getUserlanguage') },
+        { lng: 'english', value: 'en' },
+        { lng: 'german', value: 'de' },
+        { lng: 'italian', value: 'it' }
+    ];
+
+    let selected;
+
+    const onChangeValue = () => {
+        $locale && localStorage.setItem('lang', selected.value);
+        window.location.reload();
```

Comment:
Do you need to reload? Wouldn't it be possible to set the $locale to the new value?

Reply:
There is a method that can be used, I will fix this soon.